docs/content/en/docs/tasks/troubleshoot/troubleshooting.md

```diff
@@ -138,20 +125,148 @@ If you don't have any other docker containers running you may want to run `docke
+
+### Generic cluster unavailable
+Troubleshoot more by inspecting bootstrap cluster or workload cluster (depending on the stage of failure) using kubectl commands. 
+```
+kubectl get pods -A --kubeconfig=<kubeconfig>
+kubectl get nodes -A --kubeconfig=<kubeconfig>
+kubectl get logs <podname> -n <namespace> --kubeconfig=<kubeconfig>
+....
+```
+Capv troubleshooting guide: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/docs/troubleshooting.md#debugging-issues
+
+## Bare Metal troubleshooting
+
+### Bootstrap cluster fails to come up
+
+```
+Error: creating bootstrap cluster: executing create cluster: ERROR: failed to create cluster: node(s) already exist for a cluster with the name \"cluster-name\"
+, try rerunning with —force-cleanup to force delete previously created bootstrap cluster
+```
+
+The cluster already exists, so you can delete the old one:
+
+```bash
+kind delete cluster --name cluster-name
+```
+
+### Creating new workload cluster hangs or fails
+
+Cluster creation appears to be hung waiting for the Control Plane to be ready.
+If the CLI is hung on this message for over 30 mins, something likely failed during the OS provisioning:
+
+```
+Waiting for Control Plane to be ready
+```
+
+Or if cluster creation times out on this step and fails with the following messages:
+
+```
+Support bundle archive created {"path": "support-bundle-2022-06-28T00_41_24.tar.gz"}
+Analyzing support bundle {"bundle": "CLUSTER_NAME/generated/bootstrap-cluster-2022-06-28T00:41:24Z-bundle.yaml", "archive": "support-bundle-2022-06-28T00_41_24.tar.gz"}
+Analysis output generated {"path": "CLUSTER_NAME/generated/bootstrap-cluster-2022-06-28T00:43:40Z-analysis.yaml"}
+collecting workload cluster diagnostics
+Error: waiting for workload cluster control plane to be ready: executing wait: error: timed out waiting for the condition on clusters/CLUSTER_NAME
+```
+In either of those cases, the following steps can help you determine the problem:
+
+1. Export the kind cluster’s kubeconfig file:
+
+    ```bash
+    export KUBECONFIG=${PWD}/${CLUSTER_NAME}/generated/${CLUSTER_NAME}.kind.kubeconfig
+    ```
+1. If you have provided BMC information:
+
+    * Check all of the machines that the EKS Anywhere CLI has picked up from the pool of hardware in the CSV file:
+
+        ```bash
+        kubectl get machines -A
+        ```
+    * Check if those nodes are powered on. If any of those nodes are not powered on after a while then it could be possible that BMC credentials are invalid. You can verify it by checking the logs:
+
+        ```bash
+        kubectl get bmt -n eksa-system
+        kubectl get bmt <bmt-name> -n eksa-system -o yaml
+        ```
+
+    BMC credentials are incorrect if the message in the bmt log says something like `Failed to connect to BMC: failed to open connection to BMC`. You might have enable “IPMI over LAN” for that machine if it says something like “failed to perform OneTimeBootDeviceAction”
```

Comment:
@chrisdoherty4 are you suggesting something like below?
```suggestion
    Validate BMC credentials if the message in the bmt log says something like `Failed to connect to BMC: failed to open connection to BMC`. You might have enable “IPMI over LAN” for that machine if it says something like “failed to perform OneTimeBootDeviceAction”
```

Regarding "IPMI over LAN", I agree we suggest user to enable it since the beginning but this is something that can happen if they add a new machine and might forget to enable that? 
If we include IPMI command, shouldn't we have to ask them to install it as a pre-req?

Reply:
@abhinavmpandey08 Is that error consistent from every vendor? What does HP say, for example?

Reply:
```suggestion
    Validate BMC credentials are correct if a connection error is observed on the `bmt resource. Note: IPMI over LAN is required for the `bmt` resource to communicate with BMC.
```

@panktishah26 The above covers what I was referencing.

Reply:
> @abhinavmpandey08 Is that error consistent from every vendor? What does HP say, for example?

this error is specific to Dell servers. I haven't seen this error on HP or supermicro. I think those have ipmi over lan enabled by default. It's mainly the dell servers that disables by default

Reply:
@abhinavmpandey08 sure, we can add the error message for both invalid credentials and IPMI over LAN in the separate PR.
For this PR, @chrisnegus can you pl update the message suggested by Chris?

Reply:
I added the last suggestions, with a minor change. I didn't want to add a note there, so I changed the wording to say: "Note that IPMI over LAN..."